src/applications/personalization/profile/tests/components/ContactInformationField.unit.spec.jsx

```diff
@@ -94,12 +94,6 @@ describe('<ContactInformationField/>', () => {
 
     component.find('EditView').dive();
     expect(props.EditView.called).to.be.true;
-
-    const args = props.EditView.getCall(0).args[0];
-    expect(
-      args,
-      'All props were passed to the EditView constructor',
-    ).to.have.all.keys(Object.keys(props));
```

Comment:
We were checking if any and all props were passed to the EditView, but now we are only passing the props EditView really needs - making this not a very useful test that we should be able to remove.